src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -1232,7 +1179,7 @@ impl<'a, 'tcx> Visitor<'tcx> for Checker<'a, 'tcx> {
                             }
                         }
                         _ => {
-                            // In normal functions no calls are feature-gated.
+                            // Apply normal functions' rules which are not feature-gated.
```

Comment:
hmm... that doesn't seem much clearer than what we had. Maybe formulate it the other way around

```
Only in non-const functions it is perfectly fine to call any function, even ones whose constness is unstable. Everywhere else we need to check the appropriate feature gates
```